apiserver/routers/routers.go

```diff
@@ -12,8 +12,24 @@
 //    License for the specific language governing permissions and limitations
 //    under the License.
 
+// Package routers Garm API.
+//
+// The Garm API generated using go-swagger.
+//
+//	Schemes: http, https
+//	Host: localhost:9997
+//	BasePath: /api/v1
+//	Version: 1.0.0
+//	License: Apache 2.0 https://www.apache.org/licenses/LICENSE-2.0
+//
+// swagger:meta
 package routers
 
+//go:generate swagger generate spec --input=../swagger-models.yaml --output=../swagger.yaml --include="routers|controllers"
+//go:generate swagger validate ../swagger.yaml
+//go:generate rm -rf ../client
```

Comment:
can we have this live in the top level folder? `client` should be on the same level as `apiserver`

Reply:
Sure. Sounds good!